test/k8s/services.go

```diff
@@ -108,7 +108,9 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 		// KPR.
 		// See https://github.com/cilium/cilium/issues/19787
 		SkipItIf(func() bool {
-			return helpers.RunsWithKubeProxyReplacement() || helpers.SkipQuarantined()
+			return helpers.RunsWithKubeProxyReplacement() ||
+				// The test case is flaky when running on the 4.9 with k8s > 1.19.0
```

Comment:
That's oddly specific. Do we know why K8s >1.19?

Reply:
I think it might be because the dualstack IPv6 services (i.e. the ones with multiple `ipFamilies`) are provisioned only on >= 1.20. See `DualStackSupportBeta()`.

Reply:
Yep, seems likely now that I have the fix :slightly_smiling_face: : https://github.com/cilium/cilium/pull/19997.